.github/dependabot.yml

```
@@ -1,7 +1,13 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: monthly
-  open-pull-requests-limit: 10
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

```suggestion
      interval: "monthly"
  open-pull-requets-limit: 10
```
In this PR, you are removing the `open-pull-requests-limit: 10`. Is that intentional?